internal/utils.go

```diff
@@ -56,7 +57,16 @@ func FindMatches(pattern string, text []byte) map[string]interface{} {
 	r := regexp.MustCompile(pattern)
 	values := r.FindAllStringSubmatch(string(text), -1)
 	for _, match := range values {
-		configMap[match[1]] = match[2]
+		key := strings.Replace(match[1], " ", "_", -1)
```

Comment:
Not really related to this PR but it would probably make sense from now on to start adding comments to the functions that we add to `utils.go` to promote that they get reused when it's possible